charts/zabbix/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 apiVersion: v2 # Don't change this
 name: zabbix
-version: 6.0.0 # helm chart version
+version: 6.1.0 # helm chart version
```

Comment:
Hi @sinux-l5d!

Can you remove this version? I will choose the correct number during the publish process.